app/clients/models.py

```diff
@@ -12,9 +12,10 @@ def create_client(form):
 
     client_id = utils.insert('clients', client_data)
     for phone in form.data['phones']:
-        utils.insert(
-            'client_phones', {'client_id': client_id, 'phone': phone['phone']}
-        )
+        if phone['phone']:
```

Comment:
`if phone.get('phone')` mysle bezpieczniejsze. bo jezeli nie bedzie phone['phone'] to tu leci exception :>